src/openforms/submissions/tasks/emails.py

```diff
@@ -64,13 +63,6 @@ def send_confirmation_email(submission_id: int) -> None:
     if submission.confirmation_email_sent:
         return
 
-    try:
-        # access the reverse of the one2one
-        submission.form.confirmation_email_template
-    except (AttributeError, ObjectDoesNotExist):
-        logevent.confirmation_email_skip(submission)
-        return
```

Comment:
This logic is now handled in `_send_confirmation_email`.  Just mentioning it since it might look a little weird that this code is just deleted.

Reply:
Let's clean this up properly - `_send_confirmation_email` can raise a custom `SkipConfirmationEmail` exception, which the task here catches and then logs the appropriate message. I'd like to see the logging in a centralized place rather than smeared out over multiple functions, making it harder to follow what's logged where.